src/applications/personalization/dashboard-2/components/apply-for-benefits/ApplicationInProgress.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { recordDashboardClick } from '~/applications/personalization/dashboard/helpers';
+
+const ApplicationInProgress = ({
+  continueUrl,
+  expirationDate,
+  formId,
+  formTitle,
+  lastOpenedDate,
+  presentableFormId,
+}) => {
+  return (
+    <div
+      className="vads-u-display--flex vads-l-col--12 medium-screen:vads-l-col--6 small-desktop-screen:vads-l-col--4 medium-screen:vads-u-padding-right--3 vads-u-padding-bottom--3"
+      data-testid="application-in-progress"
+    >
+      <div className="vads-u-display--flex vads-u-width--full vads-u-flex-direction--column vads-u-justify-content--space-between vads-u-align-items--flex-start vads-u-background-color--gray-lightest vads-u-padding--2p5">
+        <div>
+          <div style={{ textTransform: 'uppercase' }}>{presentableFormId}</div>
+          <h4 className="vads-u-font-size--h3 vads-u-margin-top--0">
+            {formTitle}
+          </h4>
+          <div className="vads-u-display--flex">
+            <i
+              aria-hidden="true"
+              className={`fas fa-fw fa-exclamation-circle vads-u-margin-right--1 vads-u-margin-top--0p5`}
+            />
+            <div>
```

Comment:
nit: can we make this `div` a Fragment?

Reply:
nope :( it's needed to group the two `p` tags together. This `div` sits to the right of the `i` defined right before it.